VSO CSI driver documentation (#691)
hashicorp/web-unified-docs: content/vault/v1.20.x/content/docs/deploy/kubernetes/vso/index.mdx

changes made to the *source* are replicated to the *destination* over its lifetime. In this way, an application only needs
to have access to the *destination* secret in order to make use of the secret data contained within.

> For Vault Enterprise users, the Vault Secrets Operator can be alternatively deployed alongside a CSI driver to mount secrets directly to pods, 
> without making use of Kubernetes Secrets. The rest of this page covers the traditional operation of Vault Secrets Operator. 
> You can learn more about the CSI driver option on the [Vault Secrets Operator CSI driver](/vault/docs/platform/k8s/vso/vso-csi) documentation page.

Reviewer: ```suggestion

<Tip>

Vault Enterprise users can deploy the Vault Secrets Operator alongside a CSI
driver to mount secrets directly to pods without using Kubernetes Secrets.

Refer to the
[Vault Secrets Operator CSI driver](/vault/docs/platform/k8s/vso/vso-csi)
documentation to learn more about the CSI driver option.

</Tip>

```
